average.js

```diff
@@ -1,6 +1,16 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  var len = numbers.length;
+  return (
+    numbers.reduce((p, c) => {
+      if (isNaN(c)) {
```

Comment:
Nice translation of requirement into functioning code :+1: 
Your code does the summation and counting in a single pass. This has many advantages.
However, consider the other option as well: doing them in two loops would enable running in parallel, which may be faster on large datasets.